services/QuillLMS/db/structure.sql

```diff
@@ -55,6 +55,18 @@ CREATE FUNCTION public.blog_posts_search_trigger() RETURNS trigger
       $$;
 
 
+--
+-- Name: my_jsonb_to_hstore(jsonb); Type: FUNCTION; Schema: public; Owner: -
+--
+
+CREATE FUNCTION public.my_jsonb_to_hstore(jsonb) RETURNS public.hstore
```

Comment:
I believe the extra artifacts are due to the previous person dumping schema using postgres 14, rather than postgres 13.